<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboGameBackend"/>
      </item>
      <item>
       <widget class="QLabel" name="labelAddResource">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add resource to scene</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddCamera">
        <property name="text">
         <string>Add Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddLight">
        <property name="text">
         <string>Add Light</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCreateReousrce">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Create resource</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnTest">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QSpinBox" name="spinWidth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinHeight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>768</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="checkFullScreen">
          <property name="text">
           <string>Full Screen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChangeResolution">
          <property name="text">
           <string>Change Resolution</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboRenderingType"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboAntiAliasing"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboRenderTargets"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="4">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnSaveResource">
        <property name="text">
         <string>Save Resource</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDeleteResource">
        <property name="text">
         <string>Delete Resource</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnOpenResource">
        <property name="text">
         <string>Open Resource</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <widget class="QTreeWidget" name="resourceListWidget">
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Resource Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Resource Type</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QTreeWidget" name="objectListWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Object Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Object Type</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QTreeWidget" name="attributeTree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoExpandDelay">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>Attribute</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="btnRemoveObject">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionNewScene"/>
    <addaction name="actionSaveScene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuViewmode">
    <property name="title">
     <string>View mode</string>
    </property>
    <addaction name="actionWireframe"/>
    <addaction name="actionShading"/>
   </widget>
   <addaction name="menuMainMenu"/>
   <addaction name="menuViewmode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionShading">
   <property name="text">
    <string>Shading</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSaveAsProject">
   <property name="text">
    <string>Save As Project</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionNewScene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="text">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionSaveAsScene">
   <property name="text">
    <string>Save As Scene</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>exit()</slot>
 </slots>
</ui>
